app/modules/medication/index.js

```
@@ -0,0 +1,789 @@
+        if (!Number.isFinite(value) || value < 0) {
+          updateFormStatus('Bestand muss >= 0 sein.');
+          return;
+        }
+        return withBusy(target, async () => {
+          await setStock(id, Math.trunc(value), {
+            reason: 'card:set-stock',
+            dayIso: currentDayIso
+          });
+          updateFormStatus(`Bestand auf ${Math.trunc(value)} gesetzt.`);
+        });
+        return;
```

Biome (2.1.2)

[error] This code is unreachable



(lint/correctness/noUnreachable)